src/param.rs

```diff
@@ -112,6 +116,10 @@ pub enum Param {
     /// For Groups and Contacts
     ProfileImage = b'i',
 
+    /// For contacts, contains the chat id of the mailing list if this is a pseudo contacts.
+    /// If the List-Id is e.g. delta.chat.1234, then the pseudo contact is delta.chat.1234@mailing.list
```

Comment:
I completely miss the point of these pseudo-contacts. As I commented above, `mailing.list` addresses can be imported from address book, spoofed in the From field, entered by user with unexpected results and maybe security problems.

If they are used to remember that the mailing list is blocked, then we already have the ability to block groups, no need to have the mailing list as a contact.

Reply:
I also don't like this pseudo-contacts

Reply:
i think, the remaining purpose is to block/unblock mailing lists, to avoid the need to have an extra api/ui for that.

(the `mailing.list` suffix is there to make the listid a valid email address)

but maybe we can clarify that a bit more - and restrict usage to exactly that, eg. adding the pseudo-contact only if it is actually blocked. for all other purposes, adding listid to Chat seems to be better, yes.

Reply:
The pseudo contacts are to give the user a possibility to unblock the mailing list (i.e. go to `Chats and media -> Blocked contacts`, look for the `@mailing.list` contact (which is going to have the mailing list name as the display name, so easy to find) and unblock it.

If I just remove them, a user who hit "Never" on a mailing list contact request will never be able to revert this decision.

That's the only reason pseudo contacts are needed, so if we find another solution for unblocking mailing lists, we can remove them.

Reply:
What about blocked groups? We don't have any way to unblock them either.

One option is to drop these pseudo contacts and postpone group/mailinglist unblocking for a separate PR, with a proper API to list blocked chats and unblock them.

Reply:
At least on Android I didn't find a way to block a group; you can leave it, or delete it and then block all the contacts one by one when you get the contact requests.

Reply:
you cannot block groups, that's right. group blocking might be nice as well, however, this is less an issue as for mailing lists - group are usually more related to ppl you are in contact anyway. mailing lists might be pretty annoying and pop up over and over. how much this is an issue in practice is unclear, however, eg. the mentioned notifications@github.com should be blockable already today.

to move forward - yes, maybe get this out of this pr and think over that in an successor. that would make things much easier.

Reply:
`chats` table has a column "blocked", and `Chat` has `set_blocked` method. It is indeed not used now, but looks more straightforward to have API like `dc_get_blocked_chats`, `dc_block_chat`. Maybe even replace contact blocking with chat blocking in the UI, so 1:1 chat block results in a contact block?

For this PR, maybe postpone ML blocking for now and implement proper group blocking later.

Reply:
> to move forward - yes, maybe get this out of this pr and think over that in an successor. that would make things much easier.

How to we proceed for now, then? Make mailing lists blockable but not unblockable? Don't make them blockable at all but just the contacts that sent them?

Reply:
just block contacts that send them, i'd say, we can improve that in a subsequent pr, maybe even before we really do an release. also we can think about blocking groups then. however, doing all that here seems to make things hard.

Reply:
I'm ok with approving this PR with blockable but not discoverable after blocking mailing lists. This needs some solution before the release of course.